<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoShow</class>
 <widget class="QDialog" name="VideoShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>VideoShow</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>921</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="videoStartButton">
        <property name="text">
         <string>video start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoEndButton">
        <property name="text">
         <string>video end</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>exit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
